components/list/list-item-drag-drop-mixin.js

```diff
@@ -38,20 +42,16 @@ const isDragSupported = () => {
 };
 
 class DragState {
-	constructor(dragTarget) {
-		this._dragTarget = dragTarget;
+	constructor(dragTargets) {
+		this._dragTargets = dragTargets;
 		this._activeDropTarget = null;
 		this._dropTargets = new Map();
 		this._dropLocation = dropLocation.void;
 		this._time = 0;
 	}
 
-	get dragTarget() {
-		return this._dragTarget;
-	}
-
-	get dragTargetKey() {
-		return this._dragTarget && this._dragTarget.key;
+	get dragTargets() {
```

Comment:
This change is to track drag targets for the multi drag & drop case.  We are not advertising that feature just yet because we need to make some further tweaks to how to the drag image displays, but the logic is otherwise in place and works.